resolwe_bio/processes/alignment/bowtie.yml

```diff
@@ -478,7 +478,7 @@
         {% endfor %}
       {% endif %}
 
-      FW_NAME=`basename {{ reads.fastq.0.file }} .fastq.gz`
+      FW_NAME={{ reads.fastq.0.file | basename | replace(".fastq.gz", "") | replace(" ", "") }}
```

Comment:
`(reads.fastq|first).file`

Reply:
Is there any benefit of using `first` instead of `0`? Just curious. For me, `reads.fastq[0].file` is even better. Especially, considering that we would like to port this code to Python someday...

Reply:
There is no benefit. This is just an idea, you can leave it as it is.